assets/src/edit-story/app/media/utils/createResource.js

```diff
@@ -44,13 +60,15 @@ import getTypeFromMime from './getTypeFromMime';
  * resource.
  * @property {boolean} local Whether the resource has been already uploaded to
  * the server.
- * @property {Object} Object of image sizes.
+ * @property {Object} sizes Object of image sizes.
+ * @property {?Attribution} attribution An optional attribution for the
+ * resource.
  */
 
 /**
  * Creates a resource object.
  *
- * @param {Object} attachment WP Attachment object.
+ * @param {Object} an object to create the resource from.
```

Comment:
This is missing the _variable_ (`attachment` before).

```suggestion
 * @param {Object} resource An object to create the resource from.
```

Otherwise JSDoc thinks `an` is the variable name.